src/core/src/OmniTileset.cpp

```diff
@@ -277,6 +277,12 @@ void OmniTileset::addImageryIon(const pxr::SdfPath& imageryPath) {
     const OmniImagery imagery(imageryPath);
     const auto imageryIonAssetId = imagery.getIonAssetId();
     const auto imageryIonAccessToken = imagery.getIonAccessToken();
+
+    if (!imageryIonAccessToken.has_value()) {
+        // If we don't have an access token available there's no point in adding the imagery.
+        return;
```

Comment:
One open question I have is whether or not we should reprompt the user to add a default project token if no default token is available. @r-veenstra, not sure if you have any thoughts here regarding the user perspective?

Reply:
I think this would be a case where we'd want to open the token troubleshooter, though I haven't checked what Unreal and Unity do. That could be a follow up PR.